packages/3dom/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@google/3dom",
+  "version": "0.0.1",
+  "description": "An isolated scene graph facade for web-based 3D libraries",
+  "repository": "https://github.com/google/model-viewer",
+  "bugs": {
+    "url": "https://github.com/google/model-viewer/issues"
+  },
+  "homepage": "https://github.com/google/model-viewer#readme",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "author": "DevXR constributors",
+  "main": "./lib/context.js",
+  "module": "./lib/context.js",
+  "files": [
+    "src",
+    "lib"
+  ],
+  "scripts": {
+    "prepare": "if [ ! -L './shared-assets' ]; then ln -s ./node_modules/@google/model-viewer-shared-assets ./shared-assets; fi",
+    "prepack": "rm $(find ./lib -name *-spec.* -o -name test-helpers.*) 2>/dev/null",
+    "build": "tsc",
+    "clean": "rm -rf ./lib",
+    "test": "karma start --single-run",
+    "test:ci": "npm run test",
+    "watch:test": "karma start",
+    "watch:tsc": "tsc -w",
+    "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /demo/ -c-1",
+    "update:package-lock": "rm ./package-lock.json && npm i --only=production",
+    "dev": "npm run build -- --incremental && npm-run-all --parallel 'watch:tsc -- --preserveWatchOutput --incremental' 'watch:test -- --reporters dots' 'serve -- -s'"
+  },
+  "keywords": [
+    "gltf",
+    "glb",
+    "webxr",
+    "augmented reality",
+    "3d"
+  ],
+  "devDependencies": {
+    "@google/model-viewer-shared-assets": "*",
+    "@open-wc/karma-esm": "^2.11.1",
+    "@types/chai": "^4.2.7",
+    "@types/mocha": "^5.2.5",
+    "@ungap/event-target": "^0.1.0",
+    "chai": "^4.1.2",
+    "http-server": "^0.12.1",
+    "karma": "^4.0.1",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sauce-launcher": "^2.0.2",
+    "karma-sourcemap-loader": "^0.3.7",
+    "mocha": "^5.2.0",
+    "npm-run-all": "^4.1.5",
+    "polymer-build": "^3.1.1",
+    "typescript": "^3.7.5"
+  },
+  "dependencies": {
+    "three": "^0.113.1"
```

Comment:
Unfortunately, no. It's possible that we could autogenerate / patch the package.json files just before bootstrapping the repo. It would be unorthodox, but your point is well taken.